solutions/usaco-1011.mdx

```diff
@@ -15,6 +15,52 @@ find the maximum over all right triangles all in $\mathcal{O}(N^3)$ time.
 
 <LanguageSection>
  
+
+<PySection>
+
+```python
+
+from itertools import permutations
+# using itertools to generate all permutations
+
+fin = open('triangles.in', 'r')
+
+N = int(fin.readline().strip())
+points = []
+
+for _ in range(N):
+	points.append([int(p) for p in fin.readline().split()])
+
+pers = permutations(points, 3)
+
+pers = [x for x in pers]
+
+# generating all permutations of size 3, since a triangle consists of 3 points.
```

Comment:
```suggestion
pers = list(permutations(points, 3))
# generating all permutations of size 3, since a triangle consists of 3 points.
```
i'm not too good with python but i think this works?

Reply:
I submitted the solution through the USACO website, and it passes all the testcases. I'll double check for you.

Reply:
The solution looks good, just submitted again.